src/components/Composer.vue

```diff
@@ -146,16 +147,26 @@
 					<ActionButton icon="icon-folder" @click="onAddCloudAttachment">{{
 						t('mail', 'Add attachment from Files')
 					}}</ActionButton>
-					<ActionButton icon="icon-folder" @click="onAddCloudAttachmentLink">{{
+					<ActionButton :disabled="encrypt" icon="icon-folder" @click="onAddCloudAttachmentLink">{{
 						t('mail', 'Add attachment link from Files')
 					}}</ActionButton>
 					<ActionCheckbox
-						:checked="!editorPlainText"
-						:text="t('mail', 'Enable formatting')"
+						:checked="!encrypt && !editorPlainText"
+						:disabled="encrypt"
 						@check="editorPlainText = false"
 						@uncheck="editorPlainText = true"
 						>{{ t('mail', 'Enable formatting') }}</ActionCheckbox
 					>
+					<ActionCheckbox
+						v-if="mailvelopeAvailable"
+						:checked="encrypt"
+						@check="encrypt = true"
+						@uncheck="encrypt = false"
+						>{{ t('mail', 'Encrypt the message with Mailvelope.') }}</ActionCheckbox
```

Comment:
Only have this nitpick that it would be nicer and shorter like this:
> Encrypt message with Mailvelope

(No dot at end, cut "the".)

Otherwise beautiful, really great integration! :)